### ISSUE TITLE
Seems like the wrapper is not actually grabbing icons

### ISSUE THREAD

**Issue by Franco-Alejandro** [webalys-hq/streamline-wrapper-angular#2]
I've gotten into your lib files, seems like you're passing in a definition for the representation of the icon for the example on angular.... this is directly taken from the SVG. We purchased the library because it had a seemingly easy implementation on angular... 

could you explain how to properly implement the icons into a vanilla project?

**Comment by cbrwizard** [comment]
I've just installed locally `@streamlinehq/streamline-regular` package (it could've been any Streamline family package which I've purchased for my Streamline account and set the token according to our document https://www.notion.so/webalys/Using-NPM-Packages-a05deed0662b4543879271a0fc681650) in the project inside `docs/example-angular-app` app. Then I went to the family page in Streamline app https://app.streamline.to/streamline-regular in an account which has this family purchased, then I selected the first icon - a Bird House (could've been any) - and clicked `NPM Package` button in the sidebar on the right. There I could see the instructions on how to add it as code to any web project:
<img width="581" alt="Screenshot 2020-12-25 at 17 13 35" src="https://user-images.githubusercontent.com/1615659/103142645-81bc9c00-4741-11eb-9549-0443b2dfbd28.png">

I followed those and edited `docs/example-angular-app/src/app/app.component.ts` file:
```
import { Component } from '@angular/core';
import { Home } from '@streamlinehq/streamline-regular/lib/interface-essential';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
})
export class AppComponent {
  title = 'example-angular-app';
  public icon = Home.BirdHouse;
}
```

This step is described in the project's README as:
```
// To use a Streamline icon first import a category you want from a family you have access to, eg:
import { Home } from "@streamlinehq/streamline-regular/lib/interface-essential"
// And then take the icon of your choice from that category, eg with `Home.House`.
```

After this I ran the example app with `yarn start` and this is the result I've got:
<img width="508" alt="Screenshot 2020-12-25 at 17 21 09" src="https://user-images.githubusercontent.com/1615659/103142695-90f01980-4742-11eb-9e1a-968a729154fc.png">

In short, the package works as expected and following all instructions results in a proper usage. I feel like we could've made it more straightforward though. Once we have a free family which comes with a free npm package we will update all wrapper repos to use that in the examples.

Did that answer your questions?

**Comment by Franco-Alejandro** [comment]
Yeah, i mean the docs put it pretty clear to be honest.

I've set everything following the instructions for the node package and the module as well (I've got a shared module with basic libraries that are imported to each of my other modules) 

I'll attach a screenshot of my issue. Thanks for the support
![streamlineproblem](https://user-images.githubusercontent.com/29553431/103205434-4dc7af00-48d8-11eb-8576-d9d54b563883.png)

**Comment by cbrwizard** [comment]
I assume that the package is correctly installed and you're getting this error because your editor cannot find the types in the package folder.
We currently don't have types support in our icon packages. We're planning to add that in foreseeable future though.
Meanwhile here are your options:
1. Allow typescript to compile modules without types https://medium.com/@amandeepkochhar/typescript-error-could-not-find-a-declaration-file-for-module-xyz-dfbe6e45c2bd
2. Declare types in a placeholder types file https://www.detroitlabs.com/blog/2018/02/28/adding-custom-type-definitions-to-a-third-party-library/
3. Use the `Icon` type from the wrapper itself. It's declared here. https://github.com/webalys-hq/streamline-wrapper-angular/blob/master/projects/streamline-wrapper-angular/src/lib/streamline-icon.component.ts

**Comment by Franco-Alejandro** [comment]
Thanks so much for the support, I will try those options now

**Comment by Franco-Alejandro** [comment]
Ok, I've tried those... but i keep getting the same error (can't find module). I assume that the issue is on my end and not the package, but I'll get back to this thread with some more details. Thanks again!

**Comment by Franco-Alejandro** [comment]
I THINK I'VE DONE IT, thanks dude, seriously

**Comment by cbrwizard** [comment]
I'm glad to hear that :) Feel free to open a new issue if you have any!

**Comment by lucky0007** [comment]
Hi, still is not working. Please see the below screenshot.

![image](https://user-images.githubusercontent.com/12132160/103520217-cf9d7680-4e9c-11eb-9ee7-1c79ba95597c.png)

**Comment by Franco-Alejandro** [comment]
hey dude! Did you install the icons library? i had that issue at first

like, you have to, on one side, install the wrapper, and on the other the icons that you want to use

I'm using NPM so all i had to do was:

`npm install @streamlinehq/streamline-regular
`
or just add to your package.json:
`
    "@streamlinehq/streamline-regular": "^0.6.1",
`

also, I've created  a pipe for easy access to the icons through HTML, I'd suggest you do the same if you're going to use them across the project like me

**Comment by lucky0007** [comment]
@Franco-Alejandro  Thanks. It works now!
